src/styles/stopwatch.scss

```diff
@@ -0,0 +1,47 @@
+
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+  &__minutes-hand {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform-origin: bottom center;
+    transform: translate(-50%, -100%) rotate(0deg);
+    animation: rotate2 600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    position: absolute;
+    top: 10px;
+    left: 50%;
+    transform-origin: bottom center;
+    transform: translate(-50%) rotate(0deg);
+    animation: rotate 10s linear infinite;
```

Comment:
Similarly, ensure that the name of the keyframes 'rotate' is descriptive and reflects the animation's effect or purpose for better readability and maintainability.